user.go

```diff
@@ -542,6 +541,18 @@ func (user *User) unlockedConnect() {
 		}
 		go user.sendMarkdownBridgeAlert(context.TODO(), "Failed to connect to %s: %v", user.bridge.ProtocolName, err)
```

Comment:
There needs to be a return here to avoid starting the refresh timer if the connection failed

Also, the timer still doesn't get stopped if the client got logged out. I think stopping the timer in the `*messagix.Event_PermanentError` handler would be enough to handle that issue

Reply:
Good catch! Done!